docs/src/getting-involved.md

```diff
@@ -0,0 +1,104 @@
+# Getting involved
+
+It's time to get involved once you have the [fork and all required software](development.md).
+
+We assume you are working on Ubuntu (or WSL) with Visual Studio Code in this post.
+
+## Editor Plugins
+
+Install plugins below:
+
+* [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)
+* [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer)
+
+## Debugging with `launch.json`
+
+Generally, you need to debug martin with specific arguments or a config file to fix issues or add features.
+
+The most convenient way is to generate a launch.json and modify it.
+
+## Generate
+
+Press `F1` on your keyboard, and input "Generate Launch Configurations from Cargo.toml". Execute it and save it to your `.vscode` directory.
+
+## Modify
+
+Let's say you want to debugging Martin with this command:
+
+```shell
+martin postgres://postgres:postgres@localhost:5411/db --default-srid 4490
+```
+
+You could find `Debug executable 'martin'` in your `launch.json`, like below:
+
+```json
+        {
+            "type": "lldb",
+            "request": "launch",
+            "name": "Debug executable 'martin'",
+            "cargo": {
+                "args": [
+                    "build",
+                    "--bin=martin",
+                    "--package=martin"
+                ],
+                "filter": {
+                    "name": "martin",
+                    "kind": "bin"
+                }
+            },
+            "args": [],
+            "cwd": "${workspaceFolder}"
+        },
+```
+
+Just copy and paste after it, and modify your pasted like this:
+
+```json
```

Comment:
```suggestion
```yaml
```

this way comments won't be shown in red.  Possibly other language choice would also work

Reply:
Maybe `jsonc`(not sure does mdbook support it)?

Reply:
https://github.com/rust-lang/mdBook/issues/2360#issuecomment-2051968614

javascript is supported ^^

Reply:
thx! ^^ @CommanderStorm Updated to javascript already